views/index.tmpl

```diff
@@ -1,259 +1,279 @@
 {{define "content"}}
-	{{with .PageData}}
-
-	<div class="container-fluid current-next">
-	  <div class="container">
-			{{template "current_and_next" .}}
+  {{with .PageData}}
+  <div class="container-fluid current-next">
+    <div class="container">
+      {{template "current_and_next" .}}
       {{template "banner" .}}
     </div>
   </div>
-	 <div class="container-fluid index-infobar">
-   <div class="container">
-   <div class="row justify-content-center">
-        <div class="col-md-3 col-lg-3 col-xl-3">
+  <div class="container-fluid index-infobar">
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="pb-4 col-lg-3">
           <h2>Send a Message</h2>
           <hr>
           <form id="sendMessage" action="">
-          		<textarea class="form-control" rows="5" name="body">Donec id elit non mi porta gravida at eget metus. Maecenas faucibus mollis interdum.</textarea>
-          		<button type="submit" class="btn btn-primary">Send</button>
+            <textarea class="form-control" rows="5" name="body">Donec id elit non mi porta gravida at eget metus. Maecenas faucibus mollis interdum.</textarea>
+            <button type="submit" class="btn btn-primary btn-block">Send</button>
           </form>
         </div>
-
         <div class="col-lg-9">
           <h2>Get Involved</h2>
           <hr>
           <div class="row">
-          	<div class="col-lg-6">
-          		<p>University Radio York has been alive since 1967, when it became the first legal independant radio station in the UK!</p>
-          		<p>Help us continue our legacy by joining the URY team.</p>
-          		<p>Contact our teams to find out more.</p>
-          	</div>
-          	<div class="col-lg-6">
-          		<ul>
-          			<li>List of teams here.</li>
-          		</ul>
-          	</div>
-          	
+            <div class="col-lg-6">
+              <p>University Radio York has been alive since 1967, when it became the first legal independant radio station in the UK!</p>
+              <p>Help us continue our legacy by joining the URY team.</p>
+              <p>Contact our teams to find out more.</p>
+            </div>
+            <div class="col-lg-6">
+              <ul>
+                <li>List of teams here.</li>
+              </ul>
+            </div>    
           </div>
-
         </div>
       </div>
-      </div>
-      </div>
-    </div> <!-- /outer containter -->
-    
-    <div class="container outer index-schedule">
-		  <h2>Today's Schedule</h2>
-	  	<h3 style="text-align:center;">We're not broadcasting today, we'll be back on !!DATE!!.</h3>
+    </div>
+  </div>
+  <div class="container-fluid index-schedule">  
+    <div class="container">
+      <h2>Today's Schedule</h2>
+      <h3 style="text-align:center;">We're not broadcasting today, we'll be back on !!DATE!!.</h3>
```

Comment:
can be swapped for `text-center` class (but moved upwards to the container, so that the paragraph below can do the same too) :)